mainbot.py

```diff
@@ -1 +1,41 @@
 #main code for discord bot
+
+# bot.py
+import os
+import random
+
+
+
+import discord
+from dotenv import load_dotenv
+
+
+load_dotenv()
+'''TOKEN = os.getenv(DISCORD_TOKEN)'''
+
```

Comment:
use this for giving token to client.run()

Reply:
ok dear